app/services/usps_in_person_proofing/transliterable_validator.rb

```diff
@@ -0,0 +1,55 @@
+module UspsInPersonProofing
+  # Validator that can be attached to a form or other model
+  # to verify that specific supported fields are transliterable
+  #
+  # == Example
+  #
+  #   validates_with UspsInPersonProofing::TransliterableValidator,
+  #     fields: [:first_name, :last_name]
+  #
+  class TransliterableValidator < ActiveModel::Validator
+    # Initialize the validator with the given fields configured
+    # for transliteration validation
+    #
+    # @param [Hash] options
+    # @option options [Array<Symbol>] fields Fields for which to validate transliterability
+    def initialize(options)
+      super
+      @fields = options[:fields]
+      if (@fields & SUPPORTED_FIELDS).size < @fields.size
+        # Catch incorrect usage of the validator
+        raise StandardError.new(
+          "Unsupported transliteration fields: #{(@fields - SUPPORTED_FIELDS).to_json}",
+        )
+      end
```

Comment:
do we need this check ahead of time? If we added `.fetch` when we look up validations per field, we'd get errors early on that would make it clear which keys are unknown

Reply:
This is to help catch unsupported usage, so it's better that this error occur as early as possible in the development cycle. If you can point me to a mechanism for disabling this check specifically for prod, then I'd be open to adding that.